tasking/serializers/location.py

```diff
@@ -5,22 +5,43 @@
 from __future__ import unicode_literals
 
 from rest_framework import serializers
+from django.contrib.gis.geos import Point
 
 from tasking.common_tags import RADIUS_MISSING, GEODETAILS_ONLY
 from tasking.common_tags import GEOPOINT_MISSING
 from tasking.models import Location
 
 
+# pylint: disable=W0223
+class GeopointField(serializers.Field):
+    """
+    Custom Field for Geopoint
+    """
+
+    def to_internal_value(self, data):
+        """
+        Custom conversion for Geopoint field
+        """
+        geopoint = data
+
+        if geopoint is not None:
+            geopoint_split = geopoint.split(',')
+            lon = int(geopoint_split[0])
+            lat = int(geopoint_split[1])
+
+        return Point(lon, lat)
```

Comment:
Can this ever raise an exception, for example when lon and lat cannot be valid for a Point?  Can we handle such exceptions and add tests to show that we handle them?

Reply:
Tested it out. There is no case at which this can raise an exception.